src/js/components/StatusBar.js

```diff
@@ -1,64 +1,66 @@
 import React from 'react';
-
 import classNames from 'classnames';
 
 class StatusBar extends React.Component {
-  constructor() {
-    super(...arguments);
-  }
-
   static get defaultProps() {
     return {
-      className: 'progress-bar',
-      height: 10
+      className: 'progress-bar flex-box upgrade-progress-bar status-bar',
+      max: 0
```

Comment:
what if we made it so that `max` defaults to `null`, and if it is anything other than `null`, always use that no matter what?